src/main/java/org/squiddev/plethora/integration/chickens/MethodsAnalyzer.java

```diff
@@ -0,0 +1,29 @@
+package org.squiddev.plethora.integration.chickens;
+
+import com.setycz.chickens.ChickensMod;
+import org.squiddev.plethora.api.method.IContext;
+import org.squiddev.plethora.api.method.wrapper.PlethoraMethod;
+import org.squiddev.plethora.api.module.IModuleContainer;
+
+import javax.annotation.Nonnull;
+import java.util.Map;
+
+public class MethodsAnalyzer {
+
+	@PlethoraMethod(
+			module = IntegrationChickens.ANALYZER_S, modId = ChickensMod.MODID,
+			doc = "-- Get a list of all chicken species"
+	)
+	public static Map<Integer, Object> getSpeciesList(@Nonnull IContext<IModuleContainer> context) {
+		return ChickensHelper.getSpeciesListHelper(context);
```

Comment:
OK, so _technically_ you could just annotate the `ChickensHelper` method with `@PlethoraMethod` directly (or move the implementation into here). I'm not sure if that's a good idea or not though.